src/common/ZulipTextButton.js

```diff
@@ -0,0 +1,109 @@
+/* @flow strict-local */
```

Comment:
Is there a reason we can't pull in a button like this from a library of material components? I guess this is BRAND_COLOR by default, is there anything else?

Reply:
Hmm, good idea—maybe https://callstack.github.io/react-native-paper/button.html